integration_tests/integration/incidentDetailsEdit.cy.ts

```diff
@@ -148,6 +148,7 @@ context('Incident details (edit) - statement incomplete', () => {
     incidentDetailsPage.timeInputHours().clear()
     incidentDetailsPage.timeInputMinutes().clear()
     incidentDetailsPage.timeInputHours().type('13')
+    incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
```

Comment:
As this is the edit page, you should not need to select either of the yes or no radio buttons, the correct one should already be checked (the controller reads the date of discovery data on the draft adjudication which is retrieved via an API call). I think the reason these tests were failing is because you haven't added the date of discovery data into the stub for the 'stubGetDraftAdjudication' and 'stubEditDraftIncidentDetails' calls.

Reply:
removed and updated mock